packages/core/src/WorkflowExecute.ts

```diff
@@ -1318,12 +1317,12 @@ export class WorkflowExecute {
 						} catch (error) {
 							this.runExecutionData.resultData.lastNodeExecuted = executionData.node.name;
 
-							const message =
-								error instanceof ApplicationError ? error.message : OBFUSCATED_ERROR_MESSAGE;
-
+							if (!(error instanceof ApplicationError)) {
```

Comment:
since many nodes wrap unhandled errors in a `NodeOperationError` (like the notion node), we will not not be obfuscating that error.
Is that what we want to do here?

Reply:
@netroy 
we do have logic in `NodeOperationError` to determine if error should be obfuscated, if so - property `obfuscate` would be set to `true` and on FE based on that we would display `Internal error` instead actual error message

goal of this PR to obfuscate error message based on `obfuscate` property just on FE

Reply:
I'm still unsure how the error from line 1137 will ever make it Sentry considering that we catch it, but neither report it, nor rethrow it.